packages/react-native/ReactAndroid/src/main/jni/CMakeLists.txt

```diff
@@ -43,7 +43,9 @@ endfunction()
 # Third-party prefabs
 find_package(hermes-engine REQUIRED CONFIG)
 find_package(fbjni REQUIRED CONFIG)
+find_package(jsc-android REQUIRED CONFIG)
 add_library(fbjni ALIAS fbjni::fbjni)
+add_library(jsc ALIAS jsc-android::jsc)
```

Comment:
that's great. The only thing for us to remember is that we'll have to bump/publish a new version of the JSC library whenever we bump the NDK